src/index.html

```diff
@@ -8,9 +8,35 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
+    <link rel="stylesheet" href="./fonts.css">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form action="/"
+            class="search-form"
+            data-qa="big"
+            >
```

Comment:
```suggestion
      <form
        action="/"
        class="search-form"
        data-qa="big"
      >
```
Pay attention to formating and code style